taskDev3/peopleData/Printer.java

```diff
@@ -0,0 +1,70 @@
+package taskDev3.peopleData;
+
+import java.util.*;
+import taskDev3.peopleData.operations.*;
+
+/**
+ * Prints several information about people.
+ */
+class Printer {
+  private static final String SPACE = " ";
+  private static final String MEAN_AGE_MSG = "Mean people's age: ";
```

Comment:
Printer must not know implementation details of specific operation. It must be in operation, something like description